test/dfinsupp_notation.lean

```diff
@@ -7,20 +7,23 @@ example : (fun₀ | 1 | 2 | 3 => 3 | 3 => 4 : Π₀ i, Fin (i + 10)) 1 = 3 := by
 example : (fun₀ | 1 | 2 | 3 => 3 | 3 => 4 : Π₀ i, Fin (i + 10)) 2 = 3 := by simp
 example : (fun₀ | 1 | 2 | 3 => 3 | 3 => 4 : Π₀ i, Fin (i + 10)) 3 = 4 := by simp
 
-/--
-info:
--/
+/-- info: (DFinsupp.single 1 3).update 2 3 : Π₀ (i : ℕ), Fin (i + 10) -/
```

Comment:
Why did this get broken? Before it would say `fun₀ | 1 => 3 | 2 => 3`.

Reply:
Ah, I accidentally replaced `eval` with `check`. I guess `check` was already broken.

Reply:
Fixed.